go/simple/v2/examples/simple_example/simple_example.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"ampli-example/ampli"
+	"github.com/amplitude/analytics-go/amplitude"
```

Comment:
Maybe we should add aliases to `ampli.go` for some function/types to use `ampli` package name only and do not import `github.com/amplitude/analytics-go/amplitude`
```
func NewConfig(apiKey string) amplitude.Config {
	return amplitude.NewConfig(apiKey)
}

type EventOptions = amplitude.EventOptions
```

Reply:
We had a previous discussion about it [here](https://github.com/amplitude/ampli-examples/pull/132#discussion_r949816823)

Could you elaborate on that alias? 
- Do you mean to change the package name of `package ampli` at the beginning of `ampli.go`? 
- What name do you think would be appropriate?

If we change the package name to `amplipackage` for example, to use `ampli.load()` directly, users still have to first `ampli := amplipackage.Instance`

Reply:
I mean now a user should have 2 imports with similar package names `ampli` and `amplitude`:
```
import (
	"ampli-example/ampli"
)
...
ampli.Instance.Load(ampli.LoadOptions{
	Client: ampli.LoadClientOptions{
		Configuration: amplitude.NewConfig(apiKey),
	},
})
defer ampli.Instance.Shutdown()
...
ampli.Instance.Identify("user_id", identifyObj, amplitude.EventOptions{})
```
It can be confusing.

With the aliases in generated `ampli.go`:
```
func NewConfig(apiKey string) amplitude.Config {
	return amplitude.NewConfig(apiKey)
}

type EventOptions = amplitude.EventOptions
```
only `ampli` package name can be used:
```
import (
	"ampli-example/ampli"
	"github.com/amplitude/analytics-go/amplitude"
)
...
ampli.Instance.Load(ampli.LoadOptions{
	Client: ampli.LoadClientOptions{
		Configuration: ampli.NewConfig(apiKey),
	},
})
defer ampli.Instance.Shutdown()
...
ampli.Instance.Identify("user_id", identifyObj, ampli.EventOptions{})

Reply:
It's more simple and concise to import only ampli package. 

Added aliases as following and related references 
```
// NewConfig is amplitude.NewConfig alias
var NewConfig = amplitude.NewConfig

// NewClient is amplitude.NewClient alias
var NewClient = amplitude.NewClient

// EventOptions is amplitude.EventOptions alias
type EventOptions = amplitude.EventOptions
```